Backend/router/v1/report.ctrl.js

```diff
@@ -0,0 +1,67 @@
+const models = require('../../models');
+
+const create = (req, res) => {
+  models.report.create({
+    what: req.body.data[0].what,
+    location: req.body.data[0].location,
+    time: req.body.data[0].time,
+    who: req.body.data[0].who,
+    details: req.body.data[0].details,
+  }).then(result => {
+    console.log(result);
```

Comment:
Is there a reason we're logging the result in the console? Is it safe to do this?

Reply:
Just I want to see the result.

Reply:
Again, if you added the log for your testing purpose, remove this. If you're actually catching an error, or some meaningful state, created proper log messages.

Reply:
I removed it. @shberry